### ISSUE TITLE
Change database to SQL

### ISSUE THREAD

**Issue by Southclaws** [Southclaws/announce-backend#37]
v3 is pushing some metrics style time series stuff and SQL is better suited to this.

Might even use TimescaleDB!

**Comment by Bios-Marcel** [comment]
v3 nani??? Thought its canceled? Will v2 stay up?

**Comment by Southclaws** [comment]
Yes, v2 will stay up and will remain compatible despite *also* being backed by SQL - the reason I want to do this is to provide historical data for servers.
